### PR TITLE
Change vulnerabilities implicit filter field to wazuh.cluster.name

### LINKED ISSUE #6650: Change the Wazuh server filter field to wazuh.cluster.name in vulnerabilities
## Description
We need to change the field to filter the selected server Wazuh API in the vulnerabilities index-pattern, **whether it has the cluster mode enabled or not**, to `wazuh.cluster.name`.

#### Parent Issue
- https://github.com/wazuh/wazuh/issues/23316

## Tasks

- [ ] Change the field to filter the selected cluster / server to `wazuh.cluster.name`

### PR DESCRIPTION
**Pull request by asteriscos**
### Description
This pull request changes the implicit vulnerabilities filter field to wazuh.cluster.name whether the selected Wazuh server has cluster mode enabled or not.
 
### Issues Resolved
Closes #6650 

### Evidence

**Wazuh server cluster configuration**
image

**Single node mode implicit filter**
image

### Test
- Configure the Wazuh server to have cluster mode `enabled`
  - Go to Vulnerabilities **dashboard** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **inventory** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **events** and check the implicit filter uses `cluster.name` 

- Configure the Wazuh server to have cluster mode `disabled`
  - Go to Vulnerabilities **dashboard** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **inventory** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **events** and check the implicit filter uses `manager.name` 

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by JuanGarriuz** [comment]
### Test
- Configure the Wazuh server to have cluster mode `enabled`
  - Go to Vulnerabilities **dashboard** and check the implicit filter uses `wazuh.cluster.name`
image


  - Go to Vulnerabilities **inventory** and check the implicit filter uses `wazuh.cluster.name`


  - Go to Vulnerabilities **events** and check the implicit filter uses `cluster.name` 


- Configure the Wazuh server to have cluster mode `disabled`
  - Go to Vulnerabilities **dashboard** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **inventory** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **events** and check the implicit filter uses `manager.name`

**Review by JuanGarriuz — APPROVED** [review]
LGTM!!!

File: plugins/main/public/services/common-data.js [hunk 1]
```diff
@@ -160,9 +160,7 @@ export class CommonData {
             : AppState.getClusterInfo().manager,
```
Comment (Desvelao) [inline plugins/main/public/services/common-data.js:160]: review https://github.com/wazuh/wazuh-dashboard-plugins/pull/6651/files#r1593906189

File: plugins/main/public/components/common/modules/modules-helper.js [hunk 2]
```diff
@@ -165,9 +165,7 @@ If this case happens, the implicit filters are regenerated and the function is c
           ? AppState.getClusterInfo().cluster
           : AppState.getClusterInfo().manager,
```
Comment (Desvelao) [inline plugins/main/public/components/common/modules/modules-helper.js:166]: review: https://github.com/wazuh/wazuh-dashboard-plugins/pull/6651/files#r1593906189

File: plugins/main/public/components/overview/vulnerabilities/common/vulnerability_detector_adapters.tsx [hunk 3]
```diff
@@ -46,9 +46,7 @@ export const vulnerabilityIndexFiltersAdapter = (
       ? AppState.getClusterInfo().cluster
       : AppState.getClusterInfo().manager,
```
[the remark this answers is not in the document]
Comment (Desvelao) [reply plugins/main/public/components/overview/vulnerabilities/common/vulnerability_detector_adapters.tsx:47]: The indexed value is the manager name when the cluster is disabled so this is not a problem.

File: plugins/main/public/services/common-data.js (hunk 1 above)
Comment (Desvelao) [reply plugins/main/public/services/common-data.js:160]: See https://github.com/wazuh/wazuh-dashboard-plugins/pull/6651#discussion_r1593930124

File: plugins/main/public/components/common/modules/modules-helper.js (hunk 2 above)
Comment (Desvelao) [reply plugins/main/public/components/common/modules/modules-helper.js:166]: See https://github.com/wazuh/wazuh-dashboard-plugins/pull/6651#discussion_r1593930124

**Comment by Desvelao** [comment]
# Test :green_circle: 
Cluster enabled
image
image
image


Cluster disabled
image
image
image

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**